bittensor/chain_data.py

```diff
@@ -25,7 +25,7 @@
 from scalecodec.type_registry import load_type_registry_preset
 from scalecodec.utils.ss58 import ss58_encode
 
-from .utils import networking as net, U16_MAX, U16_NORMALIZED_FLOAT
+from .utils import networking as net, U16_MAX, U16_NORMALIZED_FLOAT, maybe_get_torch
```

Comment:
I know you didn't write or name the func `maybe_get_torch` but it feels like a code smell. I like how you use the env var to check for torch use. Could we update the name of `maybe_get_torch` in this PR as well?

Reply:
I could. I did think about that, and was really looking for someone else to think the same. I'll update it.